clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10163,6 +10168,9 @@ void OffloadWrapper::ConstructJob(Compilation &C, const JobAction &JA,
     WrapperArgs.push_back(
         C.getArgs().MakeArgString(Twine("-kind=") + Twine(Kind)));
 
+    if (C.getArgs().hasArg(options::OPT_fpreview_breaking_changes))
```

Comment:
What is the behavior if `fpreview-breaking-changes` is specified multiple times?

Reply:
I guess the same as if `fpreview-breaking-changes` is specified single time - it should enable preview breaking changes mode.